MedSci Rework

LeadOnTaste opened the pull request:
Пока что в основном декоративные языки и желудки, лингвистический реворк с новыми языками, робы Механикус, вендор-гардероб для робототехники Дельты, лай для E-N, ноосфера МИТ для R&D (хайвмайнд-язык при помощи импланта).

- Компилено.
- В тесте не нуждается.
- Баланс соблюдён.
- Коммьюнити одобрено.

LeadOnTaste committed:
MedSci Rework

Пока что в основном декоративные языки и желудки, лингвистический реворк с новыми языками, робы Механикус, вендор-гардероб для робототехники Дельты, лай для E-N, ноосфера МИТ для R&D (хайвмайнд-язык при помощи импланта).

Bizzonium commented:
Следует поделить коммиты логически с учётом спецификаций по наименованию

TrashDoxx commented:
Второй этап голосования не проходило, голосование администраторов и подавно.

LeadOnTaste committed:
Personal Beret

Мой личный беретик ветерана ТСФ для выдачи мне через любого педаля.

LeadOnTaste commented:
> Следует поделить коммиты логически с учётом спецификаций по наименованию

Медицину и научку и в файлах часто в один отдел объединяют, называя MedSci, из-за схожести задач и интересов.

dj-34 commented:
Ну... у нас наверное так не принято ![829695261386211350](https://user-images.githubusercontent.com/20109643/142740243-e9340f90-e157-4185-93c2-31d1bfb2a438.png)

--> https://github.com/ss220-space/Paradise/pull/471/commits/b3b684fd9b5994f44e47e1734a2cab00ffc47784

LeadOnTaste commented:
> Ну... у нас наверное так не принято ![829695261386211350](https://user-images.githubusercontent.com/20109643/142740243-e9340f90-e157-4185-93c2-31d1bfb2a438.png)
> 
> --> https://github.com/ss220-space/Paradise/pull/471/commits/b3b684fd9b5994f44e47e1734a2cab00ffc47784
> 

Всё равно входит в медицину по факту. Или точнее, в расы.

LeadOnTaste committed:
Techpriest Clothing Rework

Сделал себе персональный вариант робы техножреца и портировал спрайты с Grimdark 13, слегка изменив парочку.

LeadOnTaste committed:
Commissar Outfit

Комиссарский наряд для ГСБ, налезающий только на людей. Включает в себя тулуп и фуражку комиссара, а также файл, позволяющий напялить фуражку на Яна и получить Commissar Ian.

LeadOnTaste commented:
Закрыто на реворк
